components/Common/DarkModeToggle/index.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useEffect } from 'react';
+import ModeNightIcon from '@mui/icons-material/ModeNight';
+import BrightnessMediumIcon from '@mui/icons-material/BrightnessMedium';
+import styles from './index.module.scss';
+import { useTheme } from 'next-themes';
+
+const THEME = {
```

Comment:
Do we need a const? Isn't just a type enough? I think just a type would be enough. Can you check?

Reply:
`const` here is making the object **_readonly_** and prevent from mutating it accidentally.

Reply:
instead of writing hard coded strings , I used enums

Reply:
Not sure what you mean with "hard coded" strings. Also this is not really an "enumerator" per definition. A TypeScript type would fit better here as

`type THEME = 'DARK' | 'LIGHT'`